ROMFS/px4fmu_common/init.d/rcS

```diff
@@ -152,7 +152,7 @@ fi
 #
 if [ -f $FRC ]
 then
-	sh $FRC
+	. $FRC
```

Comment:
Can we use `source` everywhere for consistency?

Reply:
I am glad you mentioned this, sure we can but do we want to inherit the env on this call out?

Reply:
Yes the scripts currently assume that. Is that not the case on NuttX?

Reply:
`sh` is now a new process. No parent ENV changes are possible.  `.` or source are inclusions and can modify the parent.